gem/styles/gem-springster/state/state.scss

```diff
@@ -17,6 +17,9 @@
       margin-left: 10px;
     }
   }
+  #footer-bar .footer-logo {
+    width: 200px;
```

Comment:
how does this effect the smartphone version? maybe we should add max-width:200px ?